### DISCUSSION TITLE
Discord、Issues、Discussionsなどの使い分けを確認

### DISCUSSION THREAD

**Discussion by kagomen** [lgtm-factory/lgtm-factory#30] [未分類]
個人的には、

- **Issues**
  - 役割：タスクリスト
  - Issuesの消化を軸にプロジェクトが進行する
  - タスクの**内容**が現時点で明確でなくても、タスクの**存在**が確実な場合、イシューを作成してOK（忘れないために）
    - その場合はあとからイシューを編集して、タスクの内容が明確になったらそれを追記する感じ！
- **Discussions**
  - 役割：調査報告、議論
  - 記録に残しておきたい・流れてほしくないことを話したり、書き留めておく場所
  - Discordと比べて**長期間**に渡りそうな事柄を書く場所
- **Discord**
  - 役割：連絡、議論
  - Discussionsに比べて、**単発的**なことや**一方通行な連絡**を書く場所
  - 世間？に公開したくない内容を書く場所
 
という認識ですが、みなさんはどうでしょうか？
もし違う認識持ってるよーとか、説明がわかりにくいとかあれば、教えてください！！

**Comment by kazzyfrog** [comment]
自分も同じ認識です！👍

- イシューは、自分たちが扱う「問題」のリスト
- Discussions は、名前の通り、作業中の記録や質問などの、議論を行う

そして、Discussions のスレッドは、
各自作業メモなどで利用していく場合、同時に複数のスレがアクティブになります。
そして、チーム全員が、それを全て確認できるわけではないかと思います。
見落としなどあると思うので！

なので、適宜GitHub上でメンション（`@user`）するなり、
Discordで「確認・意見・ヘルプが欲しい！」などを周知する形がいいのかなと！！

**Comment by kazzyfrog** [comment]
現状で言うなら、
イシューに、
- ブランチ戦略の策定
- コミットメッセージのガイド作成

が、あったかと思います！
それぞれ、sisoさんと、kagomeさんに着手して頂いたイシュー！🌿

これは、それぞれ、
- #14 
- #22

上記にて、作業中の記録やら、相談が行われました！

ここまでは、全員が確認できるとは限らないので、
適宜気付いた人とかで、相談したり助け合って、進められればと思ってます！

自分で判断しちゃって、プルリクエストで意見をもらう形でも、もちろんOK！🧐

**Reply by kazzyfrog** [reply]
上記のイシューが完了した場合は、

- コミットメッセージの書き方
- ブランチの運用方針とブランチ名の書き方

この２つに関しては、**全員で確認・共有する必要があること**なので、
Discordにて、
「〜〜のイシューをしました！Contributing.md のドキュメントを確認してくださいー」
などのお知らせ（周知）をするのがいいのかな？と思ってます！😎

**Reply by kagomen** [reply]
たしかに、基本的にはDiscussionsは見たい人が見る、っていう感じがありますね👀
対してDiscrodは、見てくださーい！って感じですね！
